src/pages/Browse/index.spec.tsx

```diff
@@ -477,4 +477,33 @@ describe(__filename, () => {
     expect(dispatch).toHaveBeenCalledWith(fakeThunk.thunk);
     expect(_fetchVersion).toHaveBeenCalledWith({ addonId, versionId, path });
   });
+
+  it('sets a temporary page title without a version', () => {
+    const root = render();
+
+    expect(root.find('title')).toHaveText('Browse add-on version');
+  });
+
+  it('sets a page title from a version', () => {
+    const name = 'AdBlockPlus';
+    const versionString = '1.0-beta';
+    const version = {
+      ...fakeVersion,
+      id: fakeVersion.id + 1,
+      addon: {
+        ...fakeVersion.addon,
+        name: {
+          // eslint-disable-next-line @typescript-eslint/no-explicit-any
+          [(process.env as any).REACT_APP_DEFAULT_API_LANG]: name,
```

Comment:
It's a bit ugly that we have to do this. Is there a way to have a friendlier test helper that would allow us to work with localizedStrings?

Reply:
That is true. I pushed a fix, thanks.